dev/show_package_release_dates.py

```diff
@@ -6,7 +6,7 @@
 
 
 def get_distributions():
-    res = subprocess.run(["pip", "list"], stdout=subprocess.PIPE)
+    res = subprocess.run(["pip", "list"], stdout=subprocess.PIPE, check=True)
```

Comment:
Fixed this for the same reason as https://github.com/mlflow/mlflow/pull/5014/files#r744341750.